pkg/destroy/gcp/gcp.go

```diff
@@ -297,6 +340,9 @@ func (t pendingItemTracker) deletePendingItems(itemType string, items []cloudRes
 	if !exists {
 		lastFound = cloudResources{}
 	}
+	for _, item := range items {
```

Comment:
We only record the quota on deletion so we know we never double-record something.